### ISSUE TITLE
Hopper stops pulling items

### ISSUE THREAD

**Issue by Engage1955** [tom5454/Toms-Storage#99]
New at using Tom's Storage, in setting up a distribution network with 16 storage crates each attached with a hopper, after the initial pull If I add more of the same items they are not pulled. Any ideas why not?

**Comment by benbenlaw** [comment]
yeah also seeing this behaviou, it seems that it gets stuck trying to extract one item and doesnt try the next item in the inventory
